ProjectStructureAndRoles.md

```diff
@@ -75,6 +75,10 @@ CAMARA commonalities shall be processed in a Working Group “Commonalities”.
 
 All Sub Projects must comply with the work in the commonalities Working Group.
 
+**Identity and Consent Management**
+
+In the Identity and Consent Management Working Group solutions are defined and developed to identify users/subscribers, to capture, store and manage user consent, and to make sure that end user experience of using the API is not compromised.
+
```

Comment:
I assume that the Working Group “Identity and Consent Management” (as Commonalities) also works under the supervision of the TSC.

On the other hand, as Commonalities, all subprojects must be consistent (comply) with the work of the Identity and Consent Management Working Group.

Finally, we can also mention the ICM work around API access definitions and the Security and Interoperability Profile, which includes CAMARA definitions for AuthN/AuthZ flows.

These are just suggestions :)